.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1738,26 +1373,70 @@ public class FineTuningClient {
         public virtual ClientResult GetJobs(string after, int? limit, RequestOptions options);
         public virtual Task<ClientResult> GetJobsAsync(string after, int? limit, RequestOptions options);
     }
+    public class FineTuningJob {
+        public DateTimeOffset CreatedAt { get; }
+        public FineTuningJobError Error { get; }
+        public DateTimeOffset? EstimatedFinish { get; }
```

Comment:
We should customize the name of this property to be consistent, e.g. `EstimatedFinishAt` (with the 'At' suffix).

Reply:
Agree that it's inconsistent, but this is what OpenAI uses in documentation and payload. I could be persuaded if you feel strongly

Reply:
One example of established precedent for the library: we renamed `ChatCompletion.Created` to `CreatedAt`.

It's case-by-case consideration and we don't want to make changes willy-nilly, but the first priority is for the library to be usable, idiomatic, and intuitive -- well-reasoned renames and structural deviations are worthy of consideration in that context.